pax-web-runtime/src/main/java/org/ops4j/pax/web/service/internal/ServletEventDispatcher.java

```diff
@@ -135,12 +134,6 @@ public void servletEvent(final ServletEvent event) {
 		}
 		synchronized (listeners) {
 			callListeners(event);
-			Map<String, ServletEvent> events = states.get(event.getBundle());
-			if (events == null) {
-				events = new LinkedHashMap<String, ServletEvent>();
-				states.put(event.getBundle(), events);
-			}
-			events.put(event.getAlias(), event);
```

Comment:
Added that to the current code base, I think this is the most important fix for PAXWEB-803